.github/workflows/ix-cross.yml

```
@@ -0,0 +1,98 @@
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  cli-cross:
+    name: baml-cli ${{ matrix.target }}
+    # Fork PRs must never run on the persistent ix pool: the VMs are reused by
+    # later trusted, secret-bearing jobs. This lane skips rather than falling
+    # back to a GitHub-hosted runner, because it builds through the flake's
+    # cross shell and there is no nix on the Blacksmith images.
+    if: ${{ (github.event_name != 'pull_request' && github.event_name != 'pull_request_target') || github.event.pull_request.head.repo.full_name == github.repository }}
+    runs-on: [self-hosted, ix-cross]
```

actionlint (1.7.12)

[error] label "ix-cross" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows", "blacksmith-16vcpu-ubuntu-2404", "blacksmith-8vcpu-ubuntu-2404", "blacksmith-8vcpu-ubuntu-2204-arm", "blacksmith-6vcpu-macos-latest", "blacksmith-4vcpu-ubuntu-2404", "blacksmith-4vcpu-ubuntu-2204", "blacksmith-4vcpu-ubuntu-2204-arm", "blacksmith-8vcpu-windows-2025", "blacksmith-4vcpu-windows-2025", "ix". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)